dmutils/request_id.py

```diff
@@ -101,6 +101,11 @@ def init_app(app):
     app.config.setdefault("DM_SPAN_ID_HEADERS", ("X-B3-SpanId",))
     app.config.setdefault("DM_PARENT_SPAN_ID_HEADERS", ("X-B3-ParentSpan",))
 
+    # we do something a little odd here now - back-populate the first value of DM_TRACE_ID_HEADERS back to the
+    # DM_REQUEST_ID_HEADER setting, because it turns out that some apps (notably the apiclient depend on that setting
```

Comment:
Same half sentence here that I can't work out what it means.

Reply:
I didn't close my bracket.